packages/template-health-survey-ng/src/app.css

```diff
@@ -3,39 +3,62 @@ In NativeScript, the app.css file is where you place CSS rules that
 you would like to apply to your entire application. Check out
 http://docs.nativescript.org/ui/styling for a full list of the CSS
 selectors and properties you can use to style UI components.
-
 /*
 In many cases you may want to use the NativeScript core theme instead
-of writing your own CSS rules. For a full list of class names in the theme
-refer to http://docs.nativescript.org/ui/theme.
+of writing your own CSS rules. You can learn more about the 
+NativeScript core theme at https://github.com/nativescript/theme
+The imported CSS rules must precede all other types of rules.
```

Comment:
I'm not sure we should teach people CSS. In this case this is not simply good practice - imports that are not on top are invalid CSS.